manager_test.go

```diff
@@ -115,23 +116,23 @@ var _ = Describe("Extension Manager", func() {
 			Expect(eiriniManager.Options.SetupCertificateName).To(Equal("eirini-x-setupcertificate"))
 			eiriniManager.Options.SetupCertificateName = "test-setupcert"
 
-			os.RemoveAll(fmt.Sprintf("/tmp/%s", eiriniManager.Options.SetupCertificateName))
-			defer os.RemoveAll(fmt.Sprintf("/tmp/%s", eiriniManager.Options.SetupCertificateName))
+			os.RemoveAll(filepath.Join("/tmp", eiriniManager.Options.SetupCertificateName))
+			defer os.RemoveAll(filepath.Join("/tmp", eiriniManager.Options.SetupCertificateName))
 
 			Expect(eiriniManager.WebhookServer).To(BeNil())
-			Expect(afero.Exists(afero.NewOsFs(), fmt.Sprintf("/tmp/%s/tls.key", eiriniManager.Options.SetupCertificateName))).To(BeFalse())
+			Expect(afero.Exists(afero.NewOsFs(), filepath.Join("/tmp", eiriniManager.Options.SetupCertificateName, "tls.key"))).To(BeFalse())
```

Comment:
Nice! looks more neat now!  Why not using `os.TempDir()` also here?

Reply:
I... I don't know why.

Fixed.